tests/test_connection.py

```diff
@@ -244,12 +244,25 @@ def test_server_error_response(self):
 
     def test_client_error_response(self):
         request_id = get_uuid4_str()
-        error_message = 'Json node is missing child property'
+        error_message = 'Errors found processing batch update'
+        failure_messages = [
+            {
+                'index': 0,
+                'error': {
+                    'status': 'error',
+                    'message': 'Email address  is invalid'
```

Comment:
It's easier to add new items to a dictionary/list if the last element is followed by a trailing comma, could you add one here?